src/components/VideoLibraryModal.vue

```diff
@@ -532,6 +546,15 @@ const showOnScreenProgress = ref(false)
 const lastSelectedVideo = ref<VideoLibraryFile | null>(null)
 const errorProcessingVideos = ref(false)
 const deleteButtonLoading = ref(false)
+const screenWidth = computed(() => window.innerWidth)
```

Comment:
This actually won't be reactive.
I suggest using the `useWindowSize` composable from `vue-use` here.